### DISCUSSION TITLE
[Crazyflie 2.0 AI-deck Himax HM01B0 monochrome camera parameters  ]

### DISCUSSION THREAD

**Discussion by coo122567** [bitcraze/discussions#1399] [Q&A]
Hello everyone!
What are the intrinsic parameters and distortion coefficients and extrinsic parameters  of the camera on the ai-deck? I couldn't find it for a long time. I need to do camera calibration, I hope you can help!

**Comment by knmcguire** [comment]
Here is the product page of the aideck, where you can check which camera it is on the specifications: https://www.bitcraze.io/products/ai-deck/. So you can check the datasheet on that.

In terms of distortion and extrinsic parameters, I don't think we have those for you, and I think that is determined on the lens and might be slightly different for each camera. You can perhaps do the calibration yourself with this openCV function? https://docs.opencv.org/4.x/dc/dbb/tutorial_py_calibration.html

**Reply by coo122567** [reply]
Thank you! I'll give it a try, thanks for answering!!
